bucket/stretchly.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.21.1",
+    "description": "Cross-platform open source app that reminds you to take breaks",
+    "##": "Not a portable app: settings are in $Env:AppData\\stretchly\\config.json",
+    "homepage": "https://github.com/hovancik/stretchly/",
+    "license": "BSD-2-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/hovancik/stretchly/releases/download/v0.21.1/stretchly-0.21.1-win.7z",
+            "hash": "c3db8bdb10fa1f8bbb43bb08f23b25c3967b5de1100462d4b7bb8bc16efd6ab5"
+        },
+        "32bit": {
+            "url": "https://github.com/hovancik/stretchly/releases/download/v0.21.1/stretchly-0.21.1-ia32-win.7z",
+            "hash": "52d82d0ca294335809edfda25b76af03f564f4bb38ad9d9e03c43c4e8563f5b0"
+        }
+    },
+    "bin": "stretchly.exe",
+    "shortcuts": [
+        [
+            "stretchly.exe",
+            "Stretchly"
+        ]
+    ],
+    "checkver": "github",
```

Comment:
```suggestion
    "checkver": {
        "github": "https://github.com/hovancik/stretchly"
    },
```

Reply:
I force-pushed your suggestion to my branch.